README.rst

```diff
@@ -17,11 +17,14 @@
 
 .. index-start-marker
 
-Installer for D-Wave's Ocean Tools.
+`Ocean <https://docs.ocean.dwavesys.com/en/stable>`_ is
+`D-Wave's <https://www.dwavesys.com>`_ suite of tools for solving hard problems
+with quantum computers.
 
+.. index-end-marker
 
 Installation
-------------
+============
 
 .. installation-start-marker
 
```

Comment:
I would drop the "3.6+" from this line, avoid the repeated maintenance and we have the badge above if the user wants a fast way to check.

Reply:
I would also just drop the "install from source" part altogether. Partly because I believe `pip install .` is considered the preferred method these days, and partly because I don't think the target audience of the README needs this info. I think the contributors guide might be a better place for that kind of information.

Reply:
> drop the "install from source" part altogether
Are you thinking to apply this policy across all packages or in particular to the SDK?

Reply:
I would say everywhere, but the SDK especially since there isn't really a local package to install

Reply:
I always preferred pip install over setup.py, but have been including setup.py line for consistency.